### PR TITLE
Beltrami flow

### PR DESCRIPTION
**Pull request by matt-frey**

### CODE REVIEW

File: models/3d/beltrami_3d.f90 [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            x = pos(1)
+            y = pos(2)
+            z = pos(3)
+
+            cosmz = dcos(mm * z)
+            sinmz = dsin(mm * z)
+            sinkxly = dsin(kk * x + ll * y)
+            coskxly = dcos(kk * x + ll * y)
+
+            omega(1) = fk2l2 * (kk * mm * sinmz - ll * alpha * cosmz) * sinkxly
```
Comment (daviddritschel) [inline models/3d/beltrami_3d.f90:120]: Isn't this just the velocity?  The vorticity is alpha times this.

File: models/3d/beltrami_3d.f90 [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                                       dimids=dimids,                       &
+                                       varid=z_vor_id)
+
+            call close_definition(ncid)
+
+
+            kk = dble(beltrami_flow%k)
+            ll = dble(beltrami_flow%l)
+            mm = dble(beltrami_flow%m)
+            alpha = beltrami_flow%alpha
```
Comment (daviddritschel) [inline models/3d/beltrami_3d.f90:87]: Where is alpha calculated?  It must be either + or - sqrt{k^2+l^2+m^2}

File: models/3d/epic3d-models.f90 [hunk 3]
```diff
@@ -61,7 +62,7 @@ subroutine generate_fields
 
             call define_netcdf_temporal_dimension(ncid, dimids(4), axids(4))
 
-            if (model == 'TaylorGreen') then
+            if ((model == 'TaylorGreen') .or. (model == 'Beltrami')) then
```
Comment (daviddritschel) [inline models/3d/epic3d-models.f90:65]: Taylor Green should be removed - it is not a good test flow as it is very unsteady

**Review by daviddritschel — COMMENTED** [review]
Please see my comments

File: models/3d/beltrami_3d.f90 (hunk 1 above)
Comment (matt-frey) [reply models/3d/beltrami_3d.f90:120]: alpha is part of `fk2l2`.

File: models/3d/epic3d-models.f90 (hunk 3 above)
Comment (matt-frey) [reply models/3d/epic3d-models.f90:65]: I thought about removing in another PR.

**Review by daviddritschel — APPROVED** [review]

File: models/3d/epic3d-models.f90 (hunk 3 above)
Comment (daviddritschel) [reply models/3d/epic3d-models.f90:65]: Fine!

**Pull request merged**